wrappers/android/examples/sensor/src/main/java/com/intel/realsense/sensor/MainActivity.java

```diff
@@ -303,4 +234,32 @@ private synchronized void stop() {
             Log.e(TAG, "failed to stop streaming");
         }
     }
+
+    private void releaseContext() {
+        if (mDevice != null) mDevice.close();
+        if (mColorizer != null) mColorizer.close();
+
+        if(mRsContext != null) {
+            mRsContext.removeDevicesChangedCallback();
+            mRsContext.close();
+            mRsContext = null;
+        }
+    }
+
+    private void assignDepthSensor() {
+        List<Sensor> sensors = mDevice.querySensors();
+
+        for(Sensor s : sensors)
+        {
+            if (s.is(Extension.DEPTH_SENSOR)) {
+                mDepthSensor = s.as(Extension.DEPTH_SENSOR);
+            }
+        }
+    }
+    private void assignProfiles() {
+        if (mDepthSensor != null) {
+            mDepthProfile = mDepthSensor.findProfile(StreamType.DEPTH, -1, 640, 480, StreamFormat.Z16, 30);
+            mIrProfile = mDepthSensor.findProfile(StreamType.INFRARED, -1, 640, 480, StreamFormat.Y8, 30);
+        }
```

Comment:
Shouldn't you assign RGB profile here as well?

Reply:
Well, in this example, I decided to configure only the depth sensor, with depth and ir streams.